build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    Repos.addRepos(repositories)
+
+    dependencies {
+
+        classpath(Libs.android_gradle_plugin)
+        classpath(Libs.kotlin.plugin)
+        classpath(Libs.navigation.safeArgsPlugin)
+        classpath(Libs.oss.plugin)
+        classpath(Libs.kotlin.plugin)
```

Comment:
It feels like this is duplicated within this scope.

Reply:
👍